File: Regular/33/33-A.rb
#AtCoder Regular 33 A

len = STDIN.gets.chomp.to_i

num = 0

for i in 1..len
    tmp = len
    for j in 0..len
        if tmp >= i then
            num += 1
            tmp -= 1
        else
            break
        end
    end
end
puts num

File: Regular/33/33-A.c
//AtCoder Regular 33 A
#include <stdio.h>

int main( void )
{
    int length;
    int i;
    int j;
    int tmp;
    int num = 0;

    scanf( "%d" , &length );

    for( i = 1; i <= length; i++ ){
        tmp = length;
        for( j = 0; j < length; j++ ){
            if( tmp >= i ){
                num++;
                tmp--;
            }
            else{
                break;
            }
        }
    }

    printf( "%d\n" , num );

    return 0;
}
